web-ts/src/routes/DisclaimerPage.tsx

```diff
@@ -10,20 +24,70 @@ type PropsType = {
   languageModel: LanguageModel
 } & RouteComponentProps<{ cityCode: string; languageCode: string }>
 
-const DisclaimerPage = ({ match, cityModel, location }: PropsType) => {
-  const { languageCode } = match.params
+const DisclaimerPage = (props: PropsType): ReactElement => {
+  const { match, cityModel, languages, location } = props
+  const { languageCode, cityCode } = match.params
+  const viewportSmall = false
```

Comment:
Not really into this viewportSmall-Stuff, but Steffen did changes in IGAPP-679. Should your changes consider this?

Reply:
Depends on who merges first I'd say :) I can fix all existing occurrences in my PR.